web/src/js/components/Authentication/Authentication.js

```diff
@@ -65,8 +72,22 @@ import SetV4BetaMutation from 'js/mutations/SetV4BetaMutation'
 //  * we're making the username mandatory but can't rely on a field
 //    from the authentication user token to store this info
 class Authentication extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      loggedOutMessage: LOGGED_OUT_MESSAGE_TYPE.NONE,
+    }
+  }
+
   async componentDidMount() {
     await this.navigateToAuthStep()
+    const loggedOutMessage = getLoggedOutMessage()
+    if (loggedOutMessage === LOGGED_OUT_MESSAGE_TYPE.NONE) {
```

Comment:
I did this because during testing the `Authentication` component rendered twice.